src/MoBi.Presentation/Tasks/CheckNameVisitor.cs

```diff
@@ -193,7 +189,7 @@ public void Visit(IObjectBase objectBase)
 
          if (string.Equals(_oldName, objectBase.Name))
          {
-            _changes.Add(objectBase, _buildingBlock, new EditObjectBasePropertyInBuildingBlockCommand(_namePropertyName, _newName, _oldName, objectBase, _buildingBlock),
+            _changes.Add(objectBase, _buildingBlock, new RenameObjectBaseCommand(objectBase, _newName, _buildingBlock),
```

Comment:
This was the line triggered when you had that little rename helper. Formerly, the name of the building block would change, but not the name in the simulation. You'd get an indication of the change in the simulation explorer.

Now when a building block shares names with other building blocks, and you chose to rename all of them together, that will trigger renaming the same building block in all simulations too.